src/components/icon/IconWithText.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render } from '@testing-library/react'
+import IconWithText, { getFlexAlignment, getImgAlignment } from './IconWithText'
+
+describe('IconWithText', () => {
+  it('renders the children', () => {
+    const { getByText, getByAltText } = render(
+      <IconWithText
+        icon=""
+        altText="Truck Icon: A truck in motion."
+        description={'Regular shipments, scaled to demand'}
+        positionOfVisual="top"
+      />,
+    )
+
+    const label = getByText('Regular shipments, scaled to demand')
+    const image = getByAltText('Truck Icon: A truck in motion.')
+    expect(label).toBeTruthy()
+    expect(image).toBeTruthy()
+  })
+
+  it('Align content to the top or left', () => {
+    let contentAlignmentTop = getFlexAlignment('top')
+    let contentAlignmentLeft = getFlexAlignment('left')
+
+    expect(contentAlignmentTop).toBe(
+      'flex-wrap flex-col content-around w-1/2 h-40',
```

Comment:
Using [conventional comments](https://conventionalcomments.org/) style feedback:

- polish(non-blocking): Doing an exact comparison here seems brittle.  If styles unrelated to the flex alignment change, then this test will break.  In the future I'd recommend checking that the string contains the key differences between options (ie. `top => flex-col-content-around` & `left => justify-around`)